modules/fedimint-lnv2-client/src/cli.rs

```diff
@@ -28,7 +30,25 @@ pub(crate) async fn handle_cli_command(
     let opts = Opts::parse_from(iter::once(&ffi::OsString::from("lnv2")).chain(args.iter()));
 
     match opts {
-        Opts::Add { gateway } => {
+        Opts::Gateways { peer } => {
+            let gateways = match peer {
+                Some(peer) => {
+                    lightning
+                        .module_api
+                        .request_single_peer_typed::<Vec<SafeUrl>>(
```

Comment:
Seems to me we should avoid ad-hoc invocations of of API calls, and put all as logical API method on the extension trait.

Reply:
Makes sense - add and remove as well?